### PR TITLE
Add blog post for GitLab Project Integrations

### PR DESCRIPTION
**Pull request by praneetloke**

### CODE REVIEW

File: content/blog/gitlab-project-integration/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Amp-up GitLab Merge Requests With Pulumi"
+authors: ["praneet-loke"]
+tags: ["GitLab CI/CD","Pulumi"]
+date: "2020-08-26"
+meta_desc: "We are excited to announce the launch of first-class support for integrating GitLab Merge Requests with Pulumi."
+meta_image: pulumi_gitlab.png
+---
+
+We are excited to announce the launch of first-class support for integrating GitLab Merge Requests with Pulumi.
```
Comment (ericrudder) [inline content/blog/gitlab-project-integration/index.md:10]:
```suggestion
We are excited to announce the launch of native support for integrating GitLab Merge Requests with Pulumi.
```

File: content/docs/guides/continuous-delivery/gitlab-app.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+![Group Hook Setup 2](/images/docs/guides/continuous-delivery/gitlab-app/group_hook_2.png)
+
+That's it! Now when you create a merge request and run Pulumi in a merge request pipeline, you should see notes in the MR that show a summary of the Pulumi preview. Learn how to run [Pulumi in GitLab CI/CD](https://www.pulumi.com/docs/guides/continuous-delivery/gitlab-ci/).
+
+Here's a preview of what it looks like.
+
+![Merge Request Note](/images/docs/guides/continuous-delivery/gitlab-app/merge_request_note.png)
+
+## Disabling the integration
```
Comment (ericrudder) [inline content/docs/guides/continuous-delivery/gitlab-app.md:44]:
```suggestion
## Disabling the Integration
```

File: content/blog/gitlab-project-integration/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+with confidence.
+
+<!--more-->
+
+## New To Pulumi and GitLab CI/CD?
+
+GitLab's [merge request pipelines](https://docs.gitlab.com/ee/ci/merge_request_pipelines/) runs a CI/CD workflow for building and testing your software.
+Pulumi makes a great addition to it, by allowing you to preview any infrastructure changes as part of your merge request flow.
+
+Using the merge request pipeline feature together with Pulumi can help validate the changes to your infrastructure before they are applied.
```
Comment (spara) [inline content/blog/gitlab-project-integration/index.md:21]:
```suggestion
Using the merge request pipeline feature with Pulumi can validate the changes to your infrastructure before they are applied.
```

File: content/blog/gitlab-project-integration/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Once validated and approved through a peer-review process, you can apply your changes quickly and continuously with confidence while doing so;
```
Comment (spara) [inline content/blog/gitlab-project-integration/index.md:22]:
```suggestion
Once validated and approved through a peer-review process, you can apply your changes quickly and continuously with confidence while doing so,
```

File: content/blog/gitlab-project-integration/index.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+a process commonly referred to as GitOps for infrastructure.
+
+In order to leverage GitOps for your infrastructure, the Pulumi CLI offers two commands: `preview` and `up`.
```
Comment (spara) [inline content/blog/gitlab-project-integration/index.md:25]:
```suggestion
To leverage GitOps for your infrastructure, the Pulumi CLI offers two commands: `preview` and `up`.
```

File: content/blog/gitlab-project-integration/index.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+Concept | Pulumi command | Role
+--- | --- | ---
+Merge requests pipeline | `pulumi preview` | Preview infrastructure changes to be merged.
+Master branch pipeline | `pulumi up`  | Update your infrastructure with latest changes
+
+Learn how to use [Pulumi in GitLab CI/CD]({{< relref "docs/guides/continuous-delivery/gitlab-ci" >}}) with merge request pipelines as well as how to configure the [GitLab project integration with Pulumi]({{< relref "docs/guides/continuous-delivery/gitlab-app" >}}).
```
Comment (spara) [inline content/blog/gitlab-project-integration/index.md:32]:
```suggestion
Learn how to use [Pulumi in GitLab CI/CD]({{< relref "docs/guides/continuous-delivery/gitlab-ci" >}}) with merge request pipelines and how to configure the [GitLab project integration with Pulumi]({{< relref "docs/guides/continuous-delivery/gitlab-app" >}}).
```

File: content/blog/gitlab-project-integration/index.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+## Already Using Both?
+
+If you are already a user of either or both of Pulumi and GitLab CI/CD, then you probably already
```
Comment (spara) [inline content/blog/gitlab-project-integration/index.md:36]:
```suggestion
If you are already a user of either Pulumi and GitLab CI/CD, you
```

File: content/blog/gitlab-project-integration/index.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+know the importance of merge request pipelines and running Pulumi within them.
+With today's announcement you can now make any changes to your infrastructure in your merge
```
Comment (spara) [inline content/blog/gitlab-project-integration/index.md:38]:
```suggestion
With today's announcement, you can now make any changes to your infrastructure in your merge
```

File: content/blog/gitlab-project-integration/index.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+request front and center, so you don't miss any of those changes.
+
+Here's a _preview_ of what that looks like 😉
+
+![Merge Request Note](/images/docs/guides/continuous-delivery/gitlab-app/merge_request_note.png)  
+
+Head over to our docs to learn how to [configure the GitLab integration]({{< relref "docs/guides/continuous-delivery/gitlab-app" >}}) whether it's for
```
Comment (spara) [inline content/blog/gitlab-project-integration/index.md:45]:
```suggestion
Head over to our docs to learn how to [configure the GitLab integration]({{< relref "docs/guides/continuous-delivery/gitlab-app" >}}), whether for
```

File: content/blog/gitlab-project-integration/index.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+a single project or all of the projects under a GitLab Group.
+
+GitLab provides lots of tools for helping you streamline your application delivery, and the new integration with Pulumi makes things even better.
```
Comment (spara) [inline content/blog/gitlab-project-integration/index.md:48]:
```suggestion
GitLab provides lots of tools for streamlining your application delivery, and the new integration with Pulumi makes things even better.
```

File: content/blog/gitlab-project-integration/index.md (hunk 9 above)
Comment (praneetloke) [reply content/blog/gitlab-project-integration/index.md:45]: Hmm `it's` is needed here, no?

**Pull request merged**